Update intro AI monitoring copy to include a section about NVIDIA NIM (#17758)
newrelic/docs-website: src/content/docs/ai-monitoring/compatibility-requirements-ai-monitoring.mdx

    </tbody>
</table>

## Monitoring at scale with NVIDIA NIM [#deploy-at-scale]

> AI monitoring is compatible with Python and Node.js AI apps deployed with NVIDIA NIM. The integration allows you to instrument your AI-powered apps and collect data about models supported by NVIDIA NIM, even if those models aren't yet directly supported by AI monitoring. 

Author: ```suggestion
AI monitoring can integrate with NVIDIA NIM, allowing you to instrument your AI-powered apps and collect data about models even if those models aren't yet directly supported by AI monitoring. 
```

> No additional steps are needed to integrate with NVIDIA NIM: you can follow our manual procedures for installation, or you can install [directly through the New Relic platform](https://onenr.io/0VRVNLqavRa).

Author: ```suggestion
No additional steps are needed to integrate with NVIDIA NIM: you can follow our manual procedures for installation, or you can [install directly through the New Relic platform](https://onenr.io/0VRVNLqavRa).
```
```suggestion
No additional steps are needed to integrate with NVIDIA NIM: you can follow our manual procedures for installation, or you can install [directly through the New Relic platform](https://onenr.io/0VRVNLqavRa).
```
